app/org/sagebionetworks/bridge/models/healthdata/RecordExportStatusRequest.java

```diff
@@ -0,0 +1,24 @@
+package org.sagebionetworks.bridge.models.healthdata;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.sagebionetworks.bridge.json.JsonUtils;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+/**
+ * POJO to wrap synapse exporter status infomation, including a list of record ids and a exporter status
+ */
+public class RecordExportStatusRequest {
```

Comment:
Following up with Alx offline about this. It looks like there's no agreed-upon naming convention, particularly for verb-like operations. The name is fine for now, and is descriptive enough that there won't be any confusion, and can be easily changed in the future.